packages/desktop/lib/routers/actions/initialiseRouters.ts

```diff
@@ -34,12 +35,17 @@ function initialiseBaseRouters(): void {
     collectiblesRouter.set(new CollectiblesRouter())
     governanceRouter.set(new GovernanceRouter())
     initialiseBaseOnboardingRouters()
+    initialiseBaseImplicitAccountCreationRouters()
```

Comment:
`collectiblesRouter` and other subrouters are added directly, we could probably just add it here too without the need to create a function